_src/static/sass/utilities/_mixins.scss

```diff
@@ -0,0 +1,468 @@
+    @if $type == primary {
+        @extend %button-primary;
+    }
+
+    @if $type == secondary {
+        @extend %button-secondary;
+    }
+
+    @if $type == tertiary {
+        @extend %button-tertiary;
+    }
+}
+
+%button {
+    @extend %text-strong;
+    @include line-height();
+    @include box-sizing(border-box);
+    @include transition(color $timing-faster ease-in-out 0s, border-color $timing-faster ease-in-out 0s, background $timing-faster ease-in-out 0s, box-shadow $timing-faster ease-in-out 0s);
+    display: inline-block;
+    padding:($baseline / 2) $baseline;
+    border-width: 1px;
+    border-radius: ($baseline / 5);
+    border-style: solid;
+    box-shadow: none;
+    cursor: pointer;
+    text-align: center;
+
+    &:hover,
+    &:active {
+
+    }
+
+    &.disabled,
+    &[disabled],
+    &.is-disabled {
+        cursor: default;
+        pointer-events: none;
+        border: 1px solid $gray-lightest;
+        background-color: $gray-lightest;
+        color: $gray-light;
+    }
+
+    .icon-inline {
+        display: inline-block;
+        margin-right: ($baseline / 4);
+        vertical-align: middle;
+    }
+}
+
+%button-primary {
+    @extend %button;
+    background-color: $brand-primary;
+
+    &:hover,
+    &:active,
+    &:focus {
+        background-color: $brand-primary-lighter;
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+%button-secondary {
+    @extend %button;
+    background-color: $brand-secondary;
+
+    &:hover,
+    &:active,
+    &:focus {
+        background-color: $brand-secondary-lighter;
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+%button-tertiary {
+    @extend %button;
+    background-color: $white;
+    border-color: $gray-darker;
+
+    &:hover,
+    &:active,
+    &:focus {
+        box-shadow: 0 2px 1px $shadow;
+    }
+
+    &.current,
+    &.active {
+        box-shadow: inset 1px 1px 2px $shadow;
+
+        &:hover,
+        &:active,
+        &:focus {
+              box-shadow: inset 1px 1px 1px $shadow;
+        }
+    }
+}
+
+
+/* ============================
+14. Wells
+---------------------------- */
+%well {
+    box-shadow: inset 0 1px 2px 1px $shadow;
+    padding: ($baseline * 0.75) $baseline;
+}
+
+%no-content {
+    @extend %well;
+    padding: ($baseline*2);
+    background-color: $gray-lightest;
+    text-align: center;
+    color: $gray;
+
+    .button,
+    .action {
+        @extend %button-secondary;
+        margin-left: $baseline;
+
+        .icon {
+            margin-right: ($baseline / 2);
+        }
+    }
+}
+
+.is-loading {
```

Comment:
Please remove. This looks like an artifact from Studio's base styling. This is a bit specific for our base stuff now - when we get to stateful classes or cases for elements, we can add this in (and probably not here).